course_inventory/inventory.py

```diff
@@ -14,18 +14,18 @@
 from db.db_creator import DBCreator
 from environ import ENV
 from vocab import ValidDataSourceName
-from .async_enroll_gatherer import AsyncEnrollGatherer
-from .canvas_course_usage import CanvasCourseUsage
-from .gql_queries import queries as QUERIES
-from .published_date import FetchPublishedDate
+from course_inventory.async_enroll_gatherer import AsyncEnrollGatherer
+from course_inventory.canvas_course_usage import CanvasCourseUsage
+from course_inventory.gql_queries import queries as QUERIES
+from course_inventory.published_date import FetchPublishedDate
```

Comment:
Was Codacy complaining about the lack of `course_inventory` here?

Reply:
I don't think it was? From what I've seen online lately, absolute imports seem preferred over relative ones. I saw that you were doing it in `mivideo` with `queries`, so I figured I'd make the change.

Reply:
OK.  I happened to do it that way because my IDE's autocomplete feature filled that in for me.  I think I'd also read that absolute references are preferred.